routes/items/[id].tsx

```diff
@@ -7,17 +7,20 @@ import Head from "@/components/Head.tsx";
 import type { State } from "@/middleware/session.ts";
 import CommentsList from "@/islands/CommentsList.tsx";
 
-function CommentInput(props: { itemId: string }) {
+function CommentInput(props: { isLoggedIn: boolean; itemId: string }) {
```

Comment:
```suggestion
function CommentInput(props: { isSignedIn: boolean; itemId: string }) {
```
Nit: for consistency. Ditto to other instances of the variable.

Reply:
changed as proposed!